src/objects/RecurringDonationSchedule__c.object

```diff
@@ -441,6 +441,29 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
+        <fullName>IsPause__c</fullName>
```

Comment:
@sam-knox , can you please verify the description and help text for two new fields on RD Schedule?